session/src/main/scala/akka/cassandra/session/scaladsl/CassandraSession.scala

```diff
@@ -54,133 +50,32 @@ import akka.cassandra.session._
  */
 final class CassandraSession(
     system: ActorSystem,
-    sessionProvider: SessionProvider,
-    settings: CassandraSessionSettings,
+    sessionProvider: CqlSessionProvider,
     executionContext: ExecutionContext,
     log: LoggingAdapter,
     metricsCategory: String,
-    init: Session => Future[Done])
+    init: CqlSession => Future[Done])
     extends NoSerializationVerificationNeeded {
-  import settings._
 
   implicit private[akka] val ec = executionContext
   private lazy implicit val materializer = ActorMaterializer()(system)
 
-  // cache of PreparedStatement (PreparedStatement should only be prepared once)
-  private val preparedStatements =
-    new ConcurrentHashMap[String, Future[PreparedStatement]]
```

Comment:
is this built in now?

Reply:
Yep, the driver cahces prepared statements